solution-base/mongodb/Makefile

```diff
@@ -25,7 +25,10 @@ fetch-mongodb-sharded:
 		--version ${CHART_MONGO_SHARDED_VERSION} \
 		--untar \
 		--untardir ${CHART_DIR}
-
 patch:
-	@git apply --check ${PATCH_FILES}
-	@git apply ${PATCH_FILES}
+	@for patch_file in $(PATCH_FILES); do \
+		echo "Applying patch: $$patch_file"; \
+		cd $(PATCH_DIR) && git apply --3way --check "$$patch_file" || exit 1; \
+		cd $(PATCH_DIR) && git apply --3way "$$patch_file"; \
+	done
+
```

Comment:
change not needed

Reply:
https://github.com/scality/Zenko/pull/2028#discussion_r1774876326